.husky/pre-commit

```diff
@@ -1,16 +1,16 @@
-#!/usr/bin/env bash
-bash << EOF
-    STAGED_FILES_TO_LINT=$(git diff --name-only --cached --diff-filter=d -- "*.ts" "*.js")
-    STAGED_FILES_TO_ANALYZE=$(git diff --name-only --cached --diff-filter=d -- "{packages,tools}/*/src/**/!(*.css).ts")
-    STAGED_CSS_FILES=$(git diff --name-only --cached --diff-filter=d -- "{packages,tools}/**/*.css")
-    VERSION_FILE=$(dirname "$0")/../tools/base/src/version.js
+#!/bin/bash
```

Comment:
Yeah I've noticed this issue as well, however here's the documentation around why we added the bash wrapper: https://typicode.github.io/husky/how-to.html#bash

We're not currently supporting Windows so it seemed a fine compromise. The if syntax however _is_ Bash syntax so removing that would be a potential issue. What do you suggest?

Reply:
I was thinking we could migrate all these rules into a lint-staged config and just run the `yarn lint-staged` command here instead of all this logic. What are your thoughts? Was playing around with that here: https://github.com/adobe/spectrum-web-components/pull/5393

Reply:
I like the idea of migrate these in a module in a lint-staged config. More scalable. Let me update this here and test. Please feel free to add your thoughts too!